desktop/src/main/java/bisq/desktop/main/offer/offerbook/OfferBook.java

```diff
@@ -229,8 +247,8 @@ public void fillOfferBookListItems() {
             // Investigate why....
             offerBookListItems.clear();
             offerBookListItems.addAll(offerBookService.getOffers().stream()
-                    .filter(o -> !filterManager.isOfferIdBanned(o.getId()))
-                    .filter(o -> !OfferRestrictions.requiresNodeAddressUpdate() || Utils.isV3Address(o.getMakerNodeAddress().getHostName()))
+                    .filter(o -> !isOfferIdBanned(o))
+                    .filter(o -> isV3NodeAddressCompliant(o))
```

Comment:
If you use `ifOfferAllowed()` both of these filters can use method references instead. I find that's usually preferred when possible.

Reply:
Requested change in commit d709338d6366294ef727b0ab561e7ba73384872a .